index.html

```diff
@@ -0,0 +1,484 @@
+          <p class="f-font-paragraph l-flex-item l-svcs-txt-b">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+          <p class="f-font-paragraph l-flex-item l-svcs-txt-c">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a gallery of type and scrambled it to make a type specimen book.</p>
+        </section>
+        <section>
+          <div class="l-flex-container l-svcs-txt">
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">UI & UX DESIGN</h3>
+              <span class="icon-brush icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">LABORATORY</h3>
+              <span class="icon-lab icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+            <div class="l-flex-item l-svcs">
+              <h3 class="f-bold l-flex-child">DEVELOPMENT</h3>
+              <span class="icon-cog icon-big"></span>
+              <p class="l-flex-child-p">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+            </div>
+          </div>
+        </section>
+        <section>
+          <h3 class="f-bold h3-lines">Pricing Tables</h3>
+          <div class="l-flex-container">
+            <table class="l-flex-item l-flex-prices">
+              <thead>
+                <tr>
+                  <th class="f-light th-title">BASIC</th>
+                </tr></thead>
+              <tbody>
+                <tr>
+                  <td class="f-bold f-client">$5.99<span class="f-price"> / Month</span></td>
+                </tr>
+                <tr>
+                  <td>3 Projects</td>
+                </tr>
+                <tr>
+                  <td>2 Users</td>
+                </tr>
+                <tr>
+                  <td>1 Gb user</td>
+                </tr>
+                <tr>
+                  <td>1 Calendar</td>
+                </tr>
+              </tbody>
+              <tfoot>
+                <tr>
+                  <td class="td-btn"><a href=""><button class="btn f-bold">SIGN UP</button></a></td>
```

Comment:
`button`s do not go inside of anchors; use one or the other. Because this link (when active) ostensibly goes _somewhere_ (whether it's lower on the page, opening a modal, or to another page entirely), the `a` element is the most semantic.